corrector.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+class Corrector
+  def correct_name(name)
+    if name.length > 10
+      name.capitalize.strip
+    else
+      name.capitalize
+    end
+  end
+  name.capitalize[0..10]
```

Comment:
- [ ] the code in line 10 is all you need to make this method work, however, you need to make sure it is inside the correct name method, right now it is outside the method.

Reply:
Thank you for notice that.